qpc/insights/configure.py

```diff
@@ -0,0 +1,62 @@
+"""ConfigureInsightsCommand is used to set target host and port for insights."""
+from logging import getLogger
+
+import qpc.insights as insights
+from qpc import messages
+from qpc.clicommand import CliCommand
+from qpc.source.utils import validate_port
+from qpc.translation import _
+from qpc.utils import write_insights_config
+
+
+LOG = getLogger('qpc')
+
+
+# pylint: disable=too-few-public-methods
+class ConfigureInsightsCommand(CliCommand):
+    """Defines the configure host command.
+
+    This command is for configuring the target
+    host and port server for insights.
+    """
+
+    SUBCOMMAND = insights.SUBCOMMAND
+    ACTION = insights.CONFIG
+
+    def __init__(self, subparsers):
+        """Create command."""
+        # pylint: disable=no-member
+        CliCommand.__init__(self, self.SUBCOMMAND, self.ACTION,
+                            subparsers.add_parser(self.ACTION), None,
+                            None, [])
+        self.parser.add_argument('--host', dest='host', metavar='HOST',
+                                 default='console.redhat.com',
+                                 help=_(messages.INSIGHTS_CONFIG_HOST_HELP),
+                                 required=False)
+        self.parser.add_argument('--port', dest='port', metavar='PORT',
+                                 type=validate_port, default=443,
+                                 help=_(messages.INSIGHTS_CONFIG_PORT_HELP),
+                                 required=False)
+        self.parser.add_argument('--ssl-verify', dest='ssl_verify',
+                                 metavar='CERT_PATH',
+                                 help=_(messages.INSIGHTS_CONFIG_SSL_CERT_HELP),
+                                 required=False)
+        self.parser.add_argument('--use-http', dest='use_http',
+                                 action='store_true',
+                                 required=False)
+        self.parser.add_argument('--disable-auth', dest='require_token',
```

Comment:
Is there a reason why the command line option here is named differently from the internal var as well as the key in the config file?